apps/entropy-tester/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@pythnetwork/entropy-tester",
+  "version": "1.0.0",
+  "description": "Utility to test entropy provider callbacks",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
```

Comment:
Note that in modern js packages, it is recommended to use [subpath exports](https://nodejs.org/api/packages.html#subpath-exports) in addition to `main` and `types` at the root, as eventually subpath exports will be the only supported mechanism.